C/Cxx/Compat.toml

```
@@ -1,8 +1,11 @@
 ["0-0.3.1"]
 julia = "1.1"
 
-["0.3.2-0"]
+["0.3.2-0.3.3"]
 julia = "1.1.0-1"
```

I think this should be changed to
```suggestion
julia = "1.1-1.2"
```